cmd/snap-confine/snap-confine.c

```diff
@@ -277,7 +278,14 @@ static void log_startup_stage(const char *stage) {
     }
     struct timeval tv;
     gettimeofday(&tv, NULL);
-    debug("-- snap startup {\"stage\":\"%s\", \"time\":\"%lu.%06lu\"}", stage, tv.tv_sec, tv.tv_usec);
+    /* TODO ifdef on __TIMESIZE ? */
```

Comment:
It would go to the else if __TIMESIZE is not defined, right? So 32 bits. Is it really the default we want?

Reply:
tweaked to `#if __TIMESIZE == 32`